### DISCUSSION TITLE
Can't create Personal Oauth Access Tokens

### DISCUSSION THREAD

**Discussion by maneckeb** [firefly-iii/firefly-iii#9105] [Firefly III]
Hi, Please forgive me if this is a total NOOB question as I am very new to Firefly.

I am trying to create  a personal access token to utilize with the data importer.

I am on version 6.1.19 running on a shared server installed thru softalicious.  When I try to create the access token, I name it, hit create and then nothing happens.   When inspecting the window I get a POST error 500 in the console trying to find the end point/route.  Any tips to set me in the correct direction to remedy.   Everything else seems to be working fine, but I have not done much more then explore around  the app a bit.

![fireflyscreenshot](https://github.com/user-attachments/assets/6268ff62-a86e-4d3f-ba8e-167993f745db)

**Comment by JC5** [comment]
It looks like there are scripts missing. If you open the console before you try to generate a token, do you see errors already?

**Reply by maneckeb** [reply]
No errors on that specific page before I hit 'Create'.   But navigating around other links and doing other things I get 'get' and 'post' errors.   As if the links to the backend api are all bad.   The links are all pointing to public/api/...   But I have no API folder in my public directory?   Is this correct?

**Reply by JC5** [reply]
The fact that the directory doesn't exist is correct. But the fact that it links to `public` is not. This may be something in the installation, but I'm not sure. I don't know a lot about Softaculous.
